pkgs/applications/networking/cluster/argo/default.nix

```diff
@@ -2,17 +2,18 @@
 
 buildGoPackage rec {
   pname = "argo";
-  version = "2.2.1";
+  version = "2.4.0";
 
   src = fetchFromGitHub {
     owner = "argoproj";
     repo = "argo";
     rev = "v${version}";
-    sha256 = "0x3aizwbqkg2712021wcq4chmwjhw2df702wbr6zd2a2cdypwb67";
+    sha256 = "0idg0a21c0sq19202fybdwdypn6196qc7pyr9rrj3m08xq8vr6xm";
   };
 
   goDeps = ./deps.nix;
   goPackagePath = "github.com/argoproj/argo";
+  subPackages = [ "cmd/argo" ];
```

Comment:
Why switch to installing only `argo`? What about the other two commands?

Reply:
I can keep the other 2 binaries, but they are really only useful inside a Kubernetes Operator context when the full argo system is installed on a cluster.

The `argo` CLI is what I think is more relevant to users. It's more like a `argoctl`, but such a project does not exist.

I'm happy to add back the rest of the binaries if you prefer? Or I can add them back if somebody asks for them in the community?

Reply:
I'm ok to leave them off. We can add them if someone asks for them.

Reply:
another option would be to add a `argoFull` where they are available. But if the closure saving is minimal I'm okay with them being added back.